### PR TITLE
Gor/git/03 mar2021

### PR DESCRIPTION
**Pull request by Gor07s**
My draft PR

### CODE REVIEW

File: Gor/homework [hunk 1]
```diff
@@ -0,0 +1,3 @@
+1. git chechout -b branch_name - ստեղծում է նոր բռենչ ․․․ անունով
+2. git chechout -d branch_name ֊ ջնջում է ․․․ բռենչը լոկալից
```
Comment (mKolibri) [inline Gor/homework]: Are you sure, that this command delete the branch?

**Review by mKolibri — CHANGES_REQUESTED** [review]

File: Gor/homework (hunk 1 above)
Comment (Gor07s) [reply Gor/homework]: yes, it is not?

Comment (Gor07s) [reply Gor/homework]: I edit my file, pliz check it.

**Review by mKolibri — APPROVED** [review]

**Comment by mKolibri** [comment]
Hi,

I rereviewed your PR. Also marked as Approved. Please chaage PR Ready for
review and delete the branch.

Thanks,
Mane Antonyan

чт, 4 мар. 2021 г. в 00:16, Gor07s <notifications@github.com>:

**Comment by Gor07s** [comment]
Merge pull request,or whill you?

**Pull request merged**